### ISSUE TITLE
CreateShortcutModal may be an a11y violation

### ISSUE THREAD

**Issue by pascalwengerter** [owncloud/web#11097]
## Steps to reproduce
1. Look at CreateShortcutModal code 
<img width="476" alt="Screenshot 2024-06-25 at 12 03 47" src="https://github.com/owncloud/web/assets/16822008/ddf36396-7d38-4252-a878-69bafc3cc475">

2. Look at web UI 
<img width="534" alt="Screenshot 2024-06-25 at 12 04 00" src="https://github.com/owncloud/web/assets/16822008/cff039ba-5a00-4b18-bdae-4b8e760bb267">

3. Realize that using the `OcDrop` pragmatically works "in general", but may violate a screen reader since it's not a valid input and doesn't provide any context 

## Expected behavior
Creating a shortcut is possible for any user independently of any possible restrictions

## Actual behavior
A potential screenreader user gets lost in what is happening on their screen
